### PR TITLE
Use a goformation fork to go around MAX_PATH limitation 

### PR DESCRIPTION
**Pull request by ph**
The goformation library is pretty liberal with the lenght of the file it uses, this seems to mess up our windows CI because we reach the `MAX_PATH` limit.

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -1955,6 +1942,19 @@ For more information, please see http://www.bis.doc.gov
 
 See also http://www.apache.org/dev/crypto.html and/or seek legal counsel.
 
+--------------------------------------------------------------------
+Dependency: github.com/ph/goformation
```
Comment (andrewkroh) [inline NOTICE.txt]: So this change saves us 5 characters? Is that the only difference?

Comment (ph) [reply NOTICE.txt]: it's a bit more than that, It merges the cloudformation type into a single file, saving 71 chars on worst case.

**Comment by ph** [comment]
@andrewkroh functionbeat tests are now green and are executed on windows.

I also have https://github.com/elastic/beats/pull/8753 open which might not be required to do but could prevent some of the future cases.

**Comment by ph** [comment]
Backport needed for 6.x and 6.5

**Comment by ph** [comment]
🤦‍♂️ forgot the review label.

**Comment by ph** [comment]
testing

**Comment by ph** [comment]
I've misclicked and closed this PR and can't reopen it.

**Comment by ph** [comment]
@urso I move it to https://github.com/elastic/beats/pull/8859, sorry for the inconvenience.

**Pull request closed without merging**